openmc/model/model.py

```diff
@@ -1142,38 +1162,63 @@ def differentiate_depletable_mats(self, diff_volume_method: str):
             volume equally between the new materials, 'match cell' sets the
             volume of the material to volume of the cell they fill.
```

Comment:
Let's include a description of the behavior if `diff_volume_method` is `None` (no action)